osbuild/testutil/__init__.py

```diff
@@ -81,25 +81,32 @@ def mock_command(cmd_name: str, script: str):
 
 
 @contextlib.contextmanager
-def make_container(tmp_path, tag, fake_content, base="scratch"):
+def make_container(tmp_path, fake_content, base="scratch"):
     fake_container_src = tmp_path / "fake-container-src"
+    fake_container_src.mkdir(exist_ok=True)
     make_fake_tree(fake_container_src, fake_content)
     fake_containerfile_path = fake_container_src / "Containerfile"
     container_file_content = f"""
     FROM {base}
     COPY . .
     """
     fake_containerfile_path.write_text(container_file_content, encoding="utf8")
-    subprocess.check_call([
+    p = subprocess.Popen([
         "podman", "build",
         "--no-cache",
         "-f", os.fspath(fake_containerfile_path),
-        "-t", tag,
-    ])
+    ], universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
Why are we replacing this with `.Popen`? If we only want to output then we can use `subprocess.run(..., capture_output=True)` or is it important that we break on the first newline?

I'm not entirely sure here either if there's any possibility of accidentally locking up the process if it spews a lot of stderr (normally unbuffered) but it's being redirected to stdout.

Reply:
Mainly so that we keep seeing output from "podman build" while it is running. If it hangs for whatever reason or acts unexpectedly seeing the output via `pytest -s` seems nice and I couldn't think of a better way to both capture the container id (last line of the output) and to also keep streaming the output to stdout/stderr. Ideas very welcome of course :) (or maybe I'm overthinking it and `run(capture_output=True)` or `check_output()` is really enough as most of the time the container generated here is pretty small and the output of podman build therefore not that useful.

Reply:
I don't have super strong opinions on this, either one is fine and it's easier to leave as is.